benchmarks/array/cartesianProduct.bench.ts

```diff
@@ -0,0 +1,23 @@
+import * as _ from 'radashi'
+
+describe('cartesianProduct', () => {
+  bench('with no arguments', () => {
```

Comment:
I think it's kind of unnecessary to have two performance tests for “empty parameters”, I think we could just stick with []